Source/ChilliSource/Core/Scene/Scene.h

```diff
@@ -75,6 +76,19 @@ namespace ChilliSource
         /// @return Whether the system has the given interface
         //-------------------------------------------------------
         bool IsA(InterfaceIDType in_interfaceId) const override;
+        
+        /// Enable updating and rendering on the scene
+        ///
+        void Enable() noexcept { m_enabled = true; }
```

Comment:
Convention in most areas of the engine is to use SetXEnabled() rather than separate enable / disable methods. This avoids awkward things like:

    if (m_fancyEffectWhichRequiresRTTEnabled)
    {
        m_scene->Enable();
    }
    else
    {
        m_scene->Disable();
    }

reducing it to:

    m_scene->SetEnabled(m_fancyEffectWhichRequiresRTTEnabled);